index.html

```diff
@@ -276,9 +276,9 @@ <h3 class="skill-card-title">Skills</h3>
             </div>
             <!-- form -->
             <form action="https://formspree.io/f/xyyakzgb" method="POST" id="form-content">
-                <input type="text" name="FullName" max-length="30" placeholder="Full Name" class="form-row" id="ignoreindesktop" required>
-                <input type="email" name="email" max-length="30" placeholder="Email Address" id="" class="form-row" required>
-                <textarea name="comment" required id="" cols="30" rows="10" max-length="500" placeholder="Typing..." class="form-row textareas"></textarea>
+                <input type="text" name="FullName" max-length="30" placeholder="Full Name" class="form-row input" id="ignoreindesktop" required>
+                <input type="email" name="email" max-length="30" placeholder="Email Address" id="" class="form-row input" required>
+                <textarea name="comment" required id="" cols="30" rows="10" max-length="500" placeholder="Typing..." class="form-row textareas input"></textarea>
                 <div class="button-div">
                     <button type="submit" class="form-submit-button green-button"><p class="form-submit-button-text">Get in touch </p></button>
                 </div>
```

Comment:
- [ ] @francis450 , Great job implementing the local storage persistence of input values however you forgot to add `<script src="./index.js"></script>` before your closing <\body> tag to connect your index.html to your javascript file